menyer/kackerlackan/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kackerlackan</title>
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+    <header id="Header">
+        <h1>Restaurang Kackerlackan</h1>
+        <p>Välkommen till Kackerlackan, en liten hipster-pub mitt i 08:ans Södermalm, här serverar vi insekter i
+            olika former och på olika vis!</p>
+    </header>
+    <main>
+        <section id="forratter">
+            <h2>Förrätter</h2>
+
+            <div class="item">
+                <img src="assets/toastmaggot.jpg" title="En macka med larver" alt="En macka med larver" width="200">
+                <h3>Toast maggots:</h3>
+                <p>Frästa larver med en kalljäst surdegslimpa, ett trevligt alternativ om du vill byta ut
+                    toast-skagen mot något unikt!</p>
+                <p class="pris">79kr (£120)</p>
+            </div>
+            <div class="item">
+                <h3>Svartmyror med isbergsallad:</h3>
+                <img src="assets/myrsallad.jpg" alt="Sallad med svartmyror" width="200">
+                <p>Rostade svartmyror innehåller stora mängder protein och är ett mycket populärt mellanmål längst
+                    Thailands gatukök, serveras med isbergsallad och caesarsås.</p>
+                <p class="pris">57kr (£80)</p>
+            </div>
+
+        </section>
+        <section id="huvudratter">
```

Comment:
using id here makes sense since you will have 3 sections but they are all unique. Good job!